tests/components/cloud/test_tts.py

```diff
@@ -61,6 +66,12 @@ def test_default_exists() -> None:
     assert DEFAULT_TTS_DEFAULT_VOICE[1] in TTS_VOICES[DEFAULT_TTS_DEFAULT_VOICE[0]]
 
 
+def test_all_languages_have_default() -> None:
+    """Test all languages have a default voice."""
+    assert set(SUPPORT_LANGUAGES).difference(DEFAULT_VOICES) == set()
+    assert set(DEFAULT_VOICES).difference(SUPPORT_LANGUAGES) == set()
```

Comment:
Should we add a test that checks that all default voices exist in `TTS_VOICES`?

Reply:
This already do that as supported languages are constructed from it.

Reply:
I think we're only checking the languages and not the voices, right?

Reply:
Right…